script.js

```diff
@@ -1,4 +1,211 @@
+// Global variables
+var arrayActive = [];
+var scoreBoard = [];
+var numberOfRolls = 2;
+var numberOfPlayers = 2;
+var lastPlayer = 0;
+var currentPlayer = 1;
+var scoreBoardLock = 0;
+var checkWinner = "";
+
+// Game states: settings, roll dice, choose order
+var gameState = "settings";
+
+// Game settings
+var settingsList =
+  "</br>Settings: autoplay, lowest, play X</br>To start, leave input blank";
+var autoPlay = 0;
+var lowest = 0;
+
+// Main function
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  if (gameState == "settings") {
+    if (input == "autoplay") {
+      autoPlay = 1;
+      return `Autoplay enabled!${settingsList}`;
+    }
+
+    if (input == "lowest") {
+      lowest = 1;
+      return `Lowest wins enabled!${settingsList}`;
+    }
+
+    if (input.split(" ")[0] == "play") {
+      numberOfPlayers = input.split(" ")[1];
+      return `There are ${numberOfPlayers} players.${settingsList}`;
+    }
```

Comment:
Very good using of split to get the number of players! You could also have asked the player to just put the number of players but this still works